google-cloud-storage/lib/google/cloud/storage/service.rb

```diff
@@ -721,6 +721,35 @@ def update_hmac_key access_id, hmac_key_metadata_object,
           end
         end
 
+        ##
+        # Updates a bucket, including its ACL metadata.
+        def update_bucket bucket_name,
+                          bucket_gapi = nil,
+                          predefined_acl: nil,
+                          predefined_default_acl: nil,
+                          if_metageneration_match: nil,
+                          if_metageneration_not_match: nil,
+                          user_project: nil,
+                          options: {}
+          bucket_gapi ||= Google::Apis::StorageV1::Bucket.new
+          bucket_gapi.acl = [] if predefined_acl
```

Comment:
why are we setting these to empty array ? 
predefined_acl and predefined_default_acl options seems to be not used, should they be passed on to update_bucket ?

Reply:
> why are we setting these to empty array ?

 I was replicating the functionality of `patch_bucket`, so I kept the code identical. I'm not sure of the intent.
https://github.com/googleapis/google-cloud-ruby/blob/a39199b1a995d3bd32e214f5c3a8d3863aa0cc0e/google-cloud-storage/lib/google/cloud/storage/service.rb#L126-L136

Reply:
> predefined_acl and predefined_default_acl options seems to be not used, should they be passed on to update_bucket ?

Yes, I missed this. Thanks for catching it. I changed the code to pass it to apiary's update_bucket.